src/libs/components/Paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,21 @@
+import { styled } from '@mui/material/styles';
+
+export interface ParagraphProps {
+  margin?: string;
+  fontSize?: number;
+  fontWeight?: number;
+  color?: string;
+  align?: string;
+}
+
+export const Paragraph = styled('p', {
+  shouldForwardProp: prop =>
+    prop !== 'margin' && prop !== 'fontSize' && prop !== 'fontWeight' && prop !== 'color' && prop !== 'align',
+})<ParagraphProps>`
+  font-family: 'Montserrat', sans-serif;
+  margin: ${({ margin }) => (margin ? `${margin}` : '0px')};
```

Comment:
wouldn't it be better to change margin type to number?

Reply:
no I did it puprosefully